index.html

```diff
@@ -23,234 +23,278 @@
                 <div class="logo">
                     <a href="index.html">Anold</a>
                 </div>
-                <nav class="bars">
-                    <i class="fa fa-bars"></i>
+                <nav class="navbar">
+                    <div class="bars">
+                      <i class="fa fa-bars"></i>
+                    </div>
+                    <ul class="desktop_menu">
+                      <li>
+                        <a href="index.html">Portfolio</a>
+                      </li>
+                      <li>
+                        <a href="#about">About</a>
+                      </li>
```

Comment:
- [ ] **Great job incorporating the requested changes made by the previous reviewer!** However, there **is still one aspect that hasn't been addressed yet.** Your headline **is not vertically centered.** It would be great if you could adjust it to match the [provided template](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=39%3A122&mode=dev), as i**t will enhance the professionalism of your portfolio even further.**

- [ ] **[OPTIONAL]** Additionally, **if you'd like to further improve the design,** you could consider **adding a bit of `gap` between the elements.** This will bring your design closer to the template and create a more visually appealing layout. This suggestion is optional, but it could be worth considering.

   ### Your design:
   ![image](https://github.com/MPDADDY/my_portfolio/assets/107952724/2e548430-9699-4cb8-be8f-e04a9546ef3b)

   ### Figma design:
   ![image](https://github.com/MPDADDY/my_portfolio/assets/107952724/9baf7599-fcc9-4796-a6f9-d968adcb7698)